reference/customers.v2.yml

```
@@ -1254,24 +1264,32 @@ components:
                   - price
               amount:
                 type: string
-                description: A float that specifies the value applied to the price modified
-                example: '"5.0000" (Float, Float as String, Integer)'
+                description: A float that specifies the value applied to the price modified.
+                example: 5.0000" (Float, Float as String, Integer)
```

```suggestion
                example: '"5.0000"'
```
force the type clearly to string with quotes.